tools/lkl/lib/nt-host.c

```diff
@@ -288,6 +288,88 @@ struct lkl_host_operations lkl_host_ops = {
 	.jmp_buf_longjmp = jmp_buf_longjmp,
 };
 
+#ifdef PEX86_64
```

Comment:
Why is this required? Is it because of msys runs as posix? If so, shouldn't we use the full posix ops instead of nt?